vendor/k8s.io/kubernetes/staging/src/k8s.io/kube-gen/cmd/client-gen/generators/fake/generator_fake_for_type.go

```diff
@@ -420,8 +420,8 @@ var createSubresourceTemplate = `
 // Create takes the representation of a $.inputType|private$ and creates it.  Returns the server's representation of the $.resultType|private$, and an error, if there is any.
 func (c *Fake$.type|publicPlural$) Create($.type|private$Name string, $.inputType|private$ *$.inputType|raw$) (result *$.resultType|raw$, err error) {
 	obj, err := c.Fake.
-		$if .namespaced$Invokes($.NewCreateSubresourceAction|raw$($.type|allLowercasePlural$Resource, $.type|private$Name, "$.subresourcePath$", c.ns, $.inputType|private$), &$.inputType|raw${})
```

Comment:
What's the deal here?  This is fixed in upstream already?  Why not pick that one back?

Reply:
it is fixed in upstream PR, unfortunately after the downstream merged....

Reply:
@deads2k so this will be a drop on rebase because the original upstream change include this fix.

Reply:
> @deads2k so this will be a drop on rebase because the original upstream change include this fix.

ok